eslint.config.js

```diff
@@ -1,18 +1,18 @@
-import globals from 'globals';
 import eslint from '@eslint/js';
-import ts from 'typescript-eslint';
-import reactRefresh from 'eslint-plugin-react-refresh';
-import reactHooks from 'eslint-plugin-react-hooks';
-import importPlugin from 'eslint-plugin-import';
-import reactPlugin from 'eslint-plugin-react';
 
 import * as tanstackQuery from '@tanstack/eslint-plugin-query';
+import importPlugin from 'eslint-plugin-import';
+import reactPlugin from 'eslint-plugin-react';
+import reactHooks from 'eslint-plugin-react-hooks';
+import reactRefresh from 'eslint-plugin-react-refresh';
+import globals from 'globals';
+import ts from 'typescript-eslint';
 
 export default [
   eslint.configs.recommended,
   ...ts.configs.recommended,
   {
-    ignores: ['dist/**/*', 'node_modules/**/*', 'eslint.config.js'],
+    ignores: ['dist/**/*', 'node_modules/**/*', 'eslint.config.js', 'tailwind.config.js'],
```

Comment:
To be honest i don' see anything wrong to format config files with prettier. At least there will be one, consistent approach. 

I would remove albo `'eslint.config.js'`

Reply:
Don't quite understand what you mean. These changes related to eslint

Reply:
My fault. Don't know why but i assumed that this is a prettier config. Comment marked as Resolved